File: sandbox/node/src/TeamDeleteDefaultExample.ts
import * as fs from 'fs';
import api from "@dropbox/sign"
import models from "@dropbox/sign"

const apiCaller = new api.TeamApi();

apiCaller.teamDelete().catch(error => {
  console.log("Exception when calling Team#teamDelete:");
  console.log(error.body);
});

File: sandbox/dotnet/src/Dropbox.SignSandbox/TeamDeleteDefaultExample.cs
using System;
using System.Collections.Generic;
using System.IO;

using Dropbox.Sign.Api;
using Dropbox.Sign.Client;
using Dropbox.Sign.Model;

namespace Dropbox.SignSandbox;

public class TeamDeleteDefaultExample
{
    public static void Run()
    {
        var config = new Configuration();

        try
        {
            new TeamApi(config).TeamDelete();
        }
        catch (ApiException e)
        {
            Console.WriteLine("Exception when calling Team#TeamDelete: " + e.Message);
            Console.WriteLine("Status Code: " + e.ErrorCode);
            Console.WriteLine(e.StackTrace);
        }
    }
}
